### ISSUE TITLE
new item for News page : Vera Gross Lecture 

### ISSUE THREAD

**Issue by christinerogers** [3design/GlobalBrainConsortium.org#40]
Please add an item on the News page for the Vera Gross talk. 
Add new item top left;  Move all items to the right, and bump IFCN2019 to the next row. 

- [x] Title:  Vera Gross Public Lecture
- [x] image: If photo not yet available via Mary, please use this photo (re-centred for thumbnail): https://www.mghglobalpsychiatry.org/images/staff/JaninaGaller.jpg 
- [x] Subtitle: `Dr. Janina Galler delivers inaugural Vera Gross public lecture` (needs to bold and italic)
- [x] Item text:

> November 23, 2021, 12 noon EST
> 
> Registration link (free) `-> with a link on these words to:` https://www.eventbrite.ca/e/vera-gross-public-lecture-dr-janina-galler-tickets-194585920727 
> 
> Dr. Galler will present the Barbados Nutrition Study for this public presentation, made possible by a generous contribution from the Vera and Mortimer Gross Neuroinformatics Fund.
> 
> This event is will be held in conjunction with the Global Brain Consortium 2021 Annual Meeting.

- [x] Button: `Register (free)` link to : https://www.eventbrite.ca/e/vera-gross-public-lecture-dr-janina-galler-tickets-194585920727
 

(future notes -- to add when ready, later this week):
* Link to flyer 
* updated photo from Dr. Galler, via Mary

**Comment by christinerogers** [comment]
Vera Gross Public Lecture 

Registration link is public 


Button : click to view flyer

**Comment by christinerogers** [comment]
Thanks Derek,  just waiting on final review from Jessica.  will open a new issue for any further tweaks.
